docs/_Aug_milestone_.md

```diff
@@ -0,0 +1,48 @@
+## August Milestone
+
+### Overview
+
+- Focus on docs improvements
+
+### Goals
+
+- [ ] Documentation improvement
+
+  - [ ] docs: improve the structure of "Behind the scenes" sidebar + find a
+        better name #5776
+  - [ ] docs: improve the structure of "Usage scenarios" (how-to guides) sidebar
+        #5777
+  - [ ] [Spike] docs: untangle connector reference into reference, how-to guides
+        and tutorials #5961
+  - [ ] Rename "legacy juggler bridge" mentioned in docs #5558
+  - [ ] Add Couchdb connector docs #5267
+  - [ ] [APIDocs] Table not showing properly when it includes code snippet #5756
+  - [ ] Clean up extension template and docs #5336
+
+- [ ] Bugs
+
+  - [ ] includeRelations bug #4547
+
+- [ ] Miscellaneous
+
+  - [ ] CLA/DCO switch strongloop-internal/scrum-apex#436
+  - [ ] Spike: robust handling of ObjectID type for MongoDB #3456
+
+- [ ] PRs to review:
+  - [ ] feat(authentication): add support for multiple strategies on same method
+        #5735
+  - [ ] 5187 jwt refresh token Feature #5589
+  - [ ] [RFC WIP] feat(typegoose): add a component for typegoose/mongoose
+        integration #5152
```

Comment:
I would like to add https://github.com/strongloop/loopback-next/pull/5366 and https://github.com/strongloop/loopback-next/pull/5545 landed.

Reply:
added.